src/modules/Nis.rb

```diff
@@ -908,10 +908,8 @@ def Read
       @multidomain_broadcast = {} if @multidomain_broadcast == nil
       @slp_domain = {} if @slp_domain == nil
 
-      out = Convert.to_map(
-        SCR.Execute(path(".target.bash_output"), "/bin/ypdomainname")
-      )
-      @domain = Builtins.deletechars(Ops.get_string(out, "stdout", ""), "\n")
+      out = SCR.Execute(path(".target.bash_output"), "/usr/bin/ypdomainname")
+      @domain = out["stdout"].strip
```

Comment:
On one hand, It's a hostname so it should not include spaces neither at beginning or end. On the other hand is probably true that the initial intention was closer to chomp. On a third hand :wink: the 100% compatible solution would actually be ```tr("\n", "")```.